app/views/support_interface/performance/index.html.erb

```diff
@@ -98,10 +98,21 @@
 </div>
 
 <% if params[:screen] %>
-<style media="screen">
-  header, footer, .govuk-phase-banner { display: none; }
-  .govuk-width-container { max-width: 99%; }
-</style>
+  <style media="screen">
+    header, footer, .govuk-phase-banner { display: none; }
+    .govuk-width-container { max-width: 99%; }
+  </style>
 
-<meta http-equiv="refresh" content="60">
+  <meta http-equiv="refresh" content="60">
+<% else %>
+  <h2 class='govuk-heading-l'>Downloads for analysis</h2>
+
+  <h3 class='govuk-heading-m'>Application timings</h3>
+
+  <p class='govuk-body'>
+    The application timings provides data on when a application
+    form attribute was last updated by the candidate.
+  </p>
+
+  <%= govuk_link_to 'Download application timings (CSV)', support_interface_application_timings_path %>
```

Comment:
The performance page is publicly visible - do we want this to be shown if you're not logged in?

Reply:
I'd rather not show this if you're not logged in

Reply:
Should we pull out the public dashboard into a separate namespace? I half-forgot that this page was public, it's likely that something like that will happen again.

Reply:
@tijmenb 👍

Reply:
Done!